src/tests/login.js

```diff
@@ -0,0 +1,100 @@
+import chaiHttp from 'chai-http';
+import chai, { expect } from 'chai';
+import faker from 'faker';
+import models from '../models';
+import app from '../index';
+import authManager from '../helpers/jwt';
+
+chai.use(chaiHttp);
+const router = () => chai.request(app);
+
+const { hashPassword } = authManager;
+const { Users } = models;
+
+let newUser = {};
+let user = {};
+
+// get some fake data for example
+const firstName = faker.name.firstName();
+const lastName = faker.name.lastName();
+const userName = faker.internet.userName();
+const email = faker.internet.email();
+const userPassword = faker.internet.password();
+const body = {
+  firstName,
+  lastName,
+  email,
+  userName,
```

Comment:
It might be good if you call these objects from a separate file

Reply:
Okay, Will do